### PR TITLE
Setup towncrier to generate release notes

### PR DESCRIPTION
**Pull request by tmckenzie51**
### What was wrong?
Release notes process is not automated.

### How was it fixed?
Towncrier is setup in order to generate the release notes. Similar changes made as shown in the template https://github.com/ethereum/ethereum-python-project-template/pull/28#issue-302025093 

#### Cute Animal Picture

put a cute animal picture link inside the parentheses/GettyImages-634869043-58a6e83f5f9b58a3c918ca12.jpg)

### CODE REVIEW

**Comment by tmckenzie51** [comment]
@pipermerriam Should I create a docs/index.rst file similar to what's present in other repos? If so, what introduction/description/quickstart information should I include? Also for docs/release_notes.rst, I put v0.1.0-alpha.1 as the first version. Is this okay?

**Comment by tmckenzie51** [comment]
@pipermerriam Could you review this PR for me?

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="LongLine" enabled="true" level="WARNING" enabled_by_default="true" />
+  </profile>
+</component>
```
Comment (pipermerriam) [inline .idea/inspectionProfiles/Project_Default.xml:6]: Looks like you had a number of files leak in from your IDE.

File: pip-wheel-metadata/py_ecc.dist-info/LICENSE [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
```
Comment (pipermerriam) [inline pip-wheel-metadata/py_ecc.dist-info/LICENSE:22]: Some more files that look like they got committed on accident.

**Review by pipermerriam — APPROVED** [review]
Looks good other than the extra files that leaked in.  You should look into adding a [global `.gitignore`](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) which can help prevent some cases of this, though it looks like this case may have come from the changes you made to the project's `.gitignore`

Thanks for setting this up :smile_cat:

**Pull request closed without merging**